practice-app/api/views.py

```diff
@@ -18,10 +21,72 @@ def post(self, request):
             'message': f'Hello, {name}.'
         })
 
+
 class LoginAPIView(APIView):
     def post(self, *args, **kwargs):
         data = self.request.data
         serializer = ls.LoginSerializer(data=data)
         serializer.is_valid(raise_exception=True)
         return Response(serializer.data, status=st.HTTP_200_OK)
 
+
+class TotalProfitAPIView(APIView):
+    authentication_classes = (JSONWebTokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    # Helper function for calculating total profit with respect to given symbol.
+    # Default is Turkish Liras
+
+    def get_total_profit(self, username, symbol="TRY"):
+        user = User.objects.filter(username=username)[0]
```

Comment:
You can use Model.get() instead of Model.filter()[0] where exactly one item matches the query.